packages/amplify-cli/src/attach-backend.ts

```diff
@@ -73,7 +75,8 @@ async function onSuccess(context: $TSContext) {
       context.print.info(`Run 'amplify pull' to sync future upstream changes.`);
       context.print.info('');
     } else {
-      removeFolderStructure();
+      stateManager.setLocalEnvInfo(process.cwd(), { ...context.exeInfo.localEnvInfo, noUpdateBackend: true });
```

Comment:
Here upUpdateBackend is set to true, should it instead use the result in confirmKeepCodeBase ?

Reply:
Since this is in the `else` statement, `confirmKeepCodebase` will always be false. I could change the `true` to `!confirmKeepCodebase` but in my opinion that is unnecessarily obfuscating that we want to set `noUpdateBackend` to `true`.